Voycar.Api.Web/Features/Members/Endpoints/Get/Personal/Endpoint.cs

```diff
@@ -4,7 +4,7 @@ namespace Voycar.Api.Web.Features.Members.Endpoints.Get.Personal;
 using Users.Repository;
 
 
-public class Endpoint : Endpoint<Request, Results<Ok<Response>, BadRequest<NotFound>>, Mapper>
+public class Endpoint : Endpoint<Request, Results<Ok<Response>, BadRequest<BadRequest>>, Mapper>
```

Comment:
Ne den Inhalt vom Badrequest einfach ganz weg oder noch besser einen `string`, da wir ja einen Badrequest mit string zurückgeben. 
```suggestion
public class Endpoint : Endpoint<Request, Results<Ok<Response>, BadRequest<string>>, Mapper>
```